spec/getCountriesSpec.js

```diff
@@ -25,6 +25,7 @@ describe("The index", () => {
         phone_code: "+93",
         flag: "https://flagpedia.net/data/flags/h80/af.png",
         drive_direction: "right",
+        is_landlocked: true

```

Comment:
Please add comma at the end of line so other does not have to take care of line created by you 😉
```suggestion
        is_landlocked: true,
```
Same for lines (28, 44, 66, 161, 175)

Reply:
Thanks! will take care